src/unlimited.h

```diff
@@ -282,4 +282,12 @@ extern std::list<CStatBase *> mallocedStats;
 extern CCriticalSection cs_blockvalidationthread;
 void InterruptBlockValidationThreads();
 
+// Fork configuration
+/** This specifies the MTP time of the next fork */
+extern CTweak<uint64_t> miningForkTime;
+/** This specifies the minimum excessive block setting at the fork point */
+extern CTweak<uint64_t> miningForkEB;
+/** This specifies the minimum max block size at the fork point */
+extern CTweak<uint64_t> miningForkMG;
+
```

Comment:
It think this is ok to put this here for now but given that in the future we'll be doing more forks it might make sense to have a fork.h/fork.cpp where we can keep everything in one place.

Reply:
Acutally, strike that last comment...we already have uahf.h...why not just leave it there?  why move it to unlimited.h?

Reply:
I moved it because the uahf.h code specifically referred to the first fork.  I like your idea of having forks.h/forks.cpp and putting all the fork info in one file.  I'd prefer to do that (syntax-only refactor) in a subsequent PR though.

Reply:
ok fair enough.